GitHub_Learning/Feature/Dashboard/Presentation/DashboardPresenter.swift

```diff
@@ -27,26 +27,34 @@ final class DashboardPresenterImpl: BasePresenter<DashboardView> {
     
     func viewDidLoad() {
         view?.setEmail(dashboardCotroller.getEmail())
-        dashboardCotroller.subscribe(self, errorBlock: nil, updateBlock: { _ in self.updateBlock()})
+        dashboardCotroller.subscribe(self, errorBlock: nil, updateBlock: { _ in self.updateBlock() })
     }
     
     private func makeViewModelFrom(_ cryprocurrency: Cryptocurrency?) -> CryptocurrencyViewModel? {
         guard let cryprocurrencyUnwraped = cryprocurrency else { return nil }
         let url = cryprocurrencyUnwraped.imageUrl ?? ""
         let price = String(cryprocurrencyUnwraped.price ?? 0)
-        var priceChangePercentage = cryprocurrencyUnwraped.priceChange?.toString() ?? "N&N"
         let symbol = cryprocurrencyUnwraped.symbol?.uppercased() ?? "N&N"
         let priceChange: PriceChangeDirection
+        var priceChangePercentage: String
         
-        priceChangePercentage += " %"
-        if priceChangePercentage.contains("-") {
-            priceChange = .negative
-        } else if priceChangePercentage.contains("N&N") || priceChangePercentage.contains("0.0") {
-            priceChange = .neutral
-            priceChangePercentage.insert(contentsOf: "+ ", at: priceChangePercentage.startIndex)
+        if let change = cryprocurrencyUnwraped.priceChange {
+            priceChangePercentage = change.toString()
+            priceChangePercentage += " %"
```

Comment:
```suggestion
            priceChangePercentage = change.toString() + " %"
```

a nebo `"\(change.toString()) %"`